telemetry/definitions/commonDefinitions.json

```diff
@@ -6261,6 +6288,21 @@
                 }
             ]
         },
+        {
+            "name": "lambda_downloadServerlessLandPattern",
+            "description": "Called when downloading a Serverless Pattern",
+            "metadata": [
+                {
+                    "type": "action"
+                },
+                {
+                    "type": "reason"
```

Comment:
`reason` and `result` are implicitly added by default, these can be removed for all